app/models/company_value.rb

```diff
@@ -2,4 +2,5 @@
 
 class CompanyValue < ApplicationRecord
   validates :title, presence: true
+  has_many :kudos, dependent: :destroy
```

Comment:
do we really want to `destroy` all Kudos associated to the CompanyValue? I would consider `nullify` here - this is question to business, so it is not so important right now :)